### ISSUE TITLE
Get Global History

### ISSUE THREAD

**Issue by JKomoroski** [WarenGonzaga/covid19-tracker-cli#30]
Currently the `/history/:country` endpoint returns a great graph of a country's history, but I don't see a way to get the history graph for the entire world.

If this is possible I'd like to see how in the documentation.

If not, I suggest you default the `/history` endpoint to the global results.

**Comment by ianvizarra** [comment]
Hi @JKomoroski , we're communicating with the API team to provide the data for global history so we can implement it to our CLI.

**Comment by ianvizarra** [comment]
@JKomoroski We have pending PR for this implementation. For the meantime, you can test it using this temporary URL

https://covid19-tracker-c-git-fork-ianvizarra-feature-historical-33bd94.warengonzaga.now.sh/history

Eg
```bash
curl -L https://covid19-tracker-c-git-fork-ianvizarra-feature-historical-33bd94.warengonzaga.now.sh/history
```

<img width="1010" alt="Screen Shot 2020-04-05 at 1 35 22 PM" src="https://user-images.githubusercontent.com/2890710/78467746-5244aa00-7742-11ea-83c1-76f9199373ce.png">

**Comment by WarenGonzaga** [comment]
@JKomoroski this is now available in version 3.1.0
